.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+        - image: circleci/node:6
+        - image: circleci/mongo:3
+      
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run: npm install --quiet
+      - run:
+          name: Run tests
+          command: npm test
+          environment:
+            GITHUB_CLIENT_ID: test
+            GITHUB_CLIENT_SECRET: testSecret
+            SESSION_SECRET: testSessionSecret
+            GITHUB_ORG: testOrg
+      - run:
+          name: Generate coverage
+          command: |
+            npm run coverage
+            npm run coveralls
+          environment:
+            GITHUB_CLIENT_ID: test
+            GITHUB_CLIENT_SECRET: testSecret
+            SESSION_SECRET: testSessionSecret
+            GITHUB_ORG: testOrg
+            COVERALLS_SERVICE_NAME: circle-ci
+            # COVERALLS_REPO_TOKEN is a secret
+
+      - deploy:
+          command: |
```

Comment:
adding a `name:` attribute (maybe just as simple as `name: Deploy`) here would be nice, otherwise it looks kinda bad in the CircleCI dashboard:

<img width="1160" alt="screen shot 2017-09-07 at 2 18 38 pm" src="https://user-images.githubusercontent.com/697848/30181000-6d859ece-93d7-11e7-8506-400c89d09f40.png">

Reply:
Great idea